libs/model/src/services/tokenBalanceCache/providers/getSolanaBalances.ts

```diff
@@ -0,0 +1,33 @@
+import { logger } from '@hicommonwealth/logging';
+import { models } from '@hicommonwealth/model';
+import { fileURLToPath } from 'url';
+import { Balances, GetSPLBalancesOptions } from '../types';
+import { cacheBalances, getCachedBalances } from './cacheBalances';
+import { __get_spl_balances } from './get_spl_balances';
+
+const __filename = fileURLToPath(import.meta.url);
+const log = logger(__filename);
+
+export async function getSolanaBalances(
+  options: GetSPLBalancesOptions,
+  ttl?: number,
+) {
+  const cachedBalances = await getCachedBalances(options, options.addresses);
+
+  const chainNode = await models.ChainNode.scope('withPrivateData').findOne({
+    where: {
+      balance_type: 'solana',
+      name: 'Solana (Mainnet Beta)',
```

Comment:
Do we have a better way to filter than human-readable name?

Reply:
I don't believe we currently have a better way unless we create our own concept of a solana chain id. Solana uses mainnet-beta, testnet, devnet as their chain ids which are also human readable but not Ideal for UI which would be the only other way